test/framework/resource/virtualnode/manager.go

```diff
@@ -113,7 +115,9 @@ func (m *defaultManager) CheckVirtualNodeInAWS(ctx context.Context, ms *appmesh.
 	}
 	opts := equality.CompareOptionForVirtualNodeSpec()
 	if !cmp.Equal(desiredSDKVNSpec, resp.VirtualNode.Spec, opts) {
-		return errors.New(cmp.Diff(desiredSDKVNSpec, resp.VirtualNode.Spec, opts))
+		return fmt.Errorf("Observed:\n%v\n\nExpected:\n%v\nvsRefs:\n%v\nExpected VN:\n%v",
+			resp.VirtualNode.Spec.Backends, desiredSDKVNSpec.Backends, vsRefs, vn.Spec.Backends)
+		//return errors.New(cmp.Diff(desiredSDKVNSpec, resp.VirtualNode.Spec, opts))
```

Comment:
seems safe to delete the commented out code

Reply:
fixed, made a new error message just for debugging but not looking to push it as a change